### PR TITLE
[Merged by Bors] - fix: unpack all files in lake cache get

### PR DESCRIPTION
**Pull request by gebner**
Redo of #1342.  This regression was introduced in #1430.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Cache/Requests.lean [hunk 1]
```diff
@@ -33,8 +33,8 @@ def mkGetConfigContent (hashMap : IO.HashMap) : IO String := do
     pure $ (s!"url = {← mkFileURL fileName none}\n-o {IO.CACHEDIR / fileName}") :: acc
   return "\n".intercalate l
 
-/-- Calls `curl` to download files from the server to `CACHEDIR` (`.cache`) then unpacks them -/
-def getFiles (hashMap : IO.HashMap) (forceDownload : Bool) : IO Unit := do
+/-- Calls `curl` to download files from the server to `CACHEDIR` (`.cache`) -/
+def downloadFiles (hashMap : IO.HashMap) (forceDownload : Bool) : IO Unit := do
   let hashMap := if forceDownload then hashMap else hashMap.filter (← IO.getLocalCacheSet) false
```
Comment (gebner) [inline Cache/Requests.lean:38]: `hashMap` is redefined here, so `unpackCache hashMap` means something different if we call it at the end of this function.

**Comment by gebner** [comment]
bors merge

**Comment by gebner** [comment]
cc @arthurpaulino

**Comment by arthurpaulino** [comment]
Thanks for the fix! Sorry for the mix! 🙏🏼

**Pull request closed without merging**